chapters/install/os/on-rasp-linux.md

```diff
@@ -1,21 +1,64 @@
 
-# Install Jina on Raspberry Pi and other Linux Systems
+# How To Install Jina Core on Raspberry Pi OS and other Linux Systems
 
+## Overview
 
-On Raspbian or other Linux systems with Python >= 3.7 installed, you can simply install Jina via:
+This article explains how to successfully set up Jina Core on a Raspberry Pi using Raspberry Pi OS or Linux system in just a few steps.
 
-```bash
-pip install jina
-```
+## Before you start
+
+Please make sure that the following requirements are met:
+
+* Python >= 3.7 has already been installed on your system.
+
+* `pip` or `pip3` has already been installed on your system.
+
+* Setting up Jina Core requires administrative user rights.
+
+* The Jina Core software package itself requires about 12M of disk space. Furthermore, it depends on other software packages to be installed in order to function properly (see list below). Make sure that enough disk space is available on your system.
 
-On some Linux systems, PyPi may not provide the wheels on that OS. In this case, you may want to pre-install some dependencies via `apt`/`yum` not via `pip`. Since the packages on `apt`/`yum` are often pre-compiled and require much less time to install. Fortunately Jina have minimal dependencies and their corresponding `apt`/`yum` packages are:
+Fortunately, Jina Core has minimal software dependencies. The table below lists the corresponding software packages for [PyPi](https://pypi.org/) (left column), deb-based Linux distributions such as [Debian GNU/Linux](https://www.debian.org/), [Ubuntu](https://ubuntu.com/) and [Raspberry Pi OS](https://www.raspberrypi.org/software/) (middle column), and [Alpine Linux](https://alpinelinux.org/) (right column).
```

Comment:
I like it actually, I have a tendency to just ignore tables unless it explains their purpose

Reply:
Do we really need to explain right column, left column, etc though? It's pretty obvious just from looking, no?

Reply:
I put it in because the left and right column refer to a single source whereas the middle column refers to three different OS. We can take that out, of course, and simplify the sentence.

Reply:
Yeah let's take it out and  simplify the sentence

Reply:
done.